src/main/java/servlet/Calc.java

```diff
@@ -0,0 +1,117 @@
+package servlet;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+class Calc {
+    static String replaceVars(Map<String, String> vars) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4662868)